### ISSUE TITLE
Environmental yml file doesn't work on feedback.llnl.gov

### ISSUE THREAD

**Issue by zshaheen** [ACME-Climate/acme_diags#110]
I get the following using `acme_diags_env.yml`:
```
-bash-4.1$ conda env create -f acme_diags_env.yml
Using Anaconda API: https://api.anaconda.org
Fetching package metadata .................

ResolvePackageNotFound: 
  - libnetcdf 4.4.1.1 6
```

Why do people keep on deleting certain builds from Anaconda?

**Comment by chengzhuzhang** [comment]
Does it help if we remove all the builds numbers?

**Comment by zshaheen** [comment]
@chengzhuzhang That's what I'm trying right now.

**Comment by zshaheen** [comment]
Tested and fixed in 08cbcd8199c46a44736b2ae9158a7655846db54b
